paimon-hive/paimon-hive-catalog/pom.xml

```diff
@@ -112,6 +112,35 @@ under the License.
             </exclusions>
         </dependency>
 
+        <!-- for hudi clone -->
+        <dependency>
+            <groupId>org.apache.hudi</groupId>
+            <artifactId>hudi-common</artifactId>
```

Comment:
Can you just create a model for hudi? for example, paimon-hudi? And in hive catalog we can use java spi to load hudi files factory.